src/ShopBundle/objects/db/TShopOrderStep/TShopStepUserDataV2EndPoint.class.php

```diff
@@ -425,26 +425,37 @@ protected function ValidateStepData()
      * will be registered.
      *
      * @param array $aUserData
+     * @return bool
      */
     protected function UpdateUser($aUserData)
```

Comment:
Are there any other known implementers of this method (in sub classes)?

Reply:
Not in core, no.

There are however a couple of projects I know of that overwrite it in "userspace" - which is why I was careful to explicitly check for `false` in order to not introduce a bc-break.